bindings/ocaml/analysis/llvm_analysis.mli

```diff
@@ -44,3 +44,18 @@ external view_function_cfg : Llvm.llvalue -> unit = "llvm_view_function_cfg"
     See [llvm::Function::viewCFGOnly]. *)
 external view_function_cfg_only : Llvm.llvalue -> unit
                                 = "llvm_view_function_cfg_only"
+(* [get_callgraph m] gets the call graph of module [m] *)
```

Comment:
```suggestion

(** [get_callgraph m] gets the call graph of module [m] *)
```

Reply:
it would be good to add 'See [llvm::...]' with the appropriate methods here and below